packages/kbn-eslint-plugin-eslint/rules/no_route_security_defined.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+module.exports = {
```

Comment:
Eslint draft rule to autofix routes missing authorization config. This could be potentially used to migrate the codebase to new approach.